po/tr.po

```diff
@@ -0,0 +1,172 @@
+# Turkish translations for io.github.tfuxu.floodit package.
+# Copyright (C) 2024 io.github.tfuxu.floodit's COPYRIGHT HOLDER
+# This file is distributed under the same license as the io.github.tfuxu.floodit package.
+#
+# Sabri Ünal <yakushabb@gmail.com>, 2024.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: io.github.tfuxu.floodit\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2024-11-06 00:30+0300\n"
+"PO-Revision-Date: 2024-11-06 00:37+0300\n"
+"Last-Translator: Sabri Ünal <yakushabb@gmail.com>\n"
+"Language-Team: Türkçe <takim@gnome.org.tr>\n"
+"Language: tr\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=1; plural=0;\n"
+"X-Generator: Poedit 3.5\n"
+
+#: data/io.github.tfuxu.floodit.metainfo.xml.in.in:4
+#: data/io.github.tfuxu.floodit.desktop.in.in:7 data/ui/main_window.blp:5
+msgid "Flood It"
+msgstr "Flood It"
+
+#: data/io.github.tfuxu.floodit.metainfo.xml.in.in:5
+#: data/io.github.tfuxu.floodit.desktop.in.in:8 data/ui/main_window.blp:31
+msgid "Flood the board"
+msgstr "Panoyu doldur"
+
+#: data/io.github.tfuxu.floodit.metainfo.xml.in.in:17
+msgid ""
+"Flood It is a game with the simple premise of flooding the entire board with "
+"one color in the least amount of moves possible. Challenge yourself with "
+"this simple, yet addictive strategy game, where you need to flood-it as "
+"efficiently as you can!"
+msgstr ""
+"Flood It, mümkün olan en az hamlede tüm panoyu tek renkle doldurmanızı "
+"isteyen eğlenceli bir oyundur. Basit fakat bağımlılık yapan strateji "
+"oyunuyla kendinize meydan okurken olabildiğince verimli şekilde panoyu "
+"renklere boyayın!"
+
+#. Translators: These are search terms to find this application. Do NOT translate or localize the semicolons. The list MUST also end with a semicolon.
+#: data/io.github.tfuxu.floodit.desktop.in.in:12
+msgid "game;strategy;flood;flood-it;"
+msgstr "game;strategy;flood;flood-it;oyun;strateji;sel;baskın;taşkın;"
```

Comment:
Shouldn't keywords in English be removed?

Reply:
Due to a glib error, translators should keep the existing keywords intact and add new keywords afterward.

Reply:
I see, thanks for clarification! BTW, to you have a link to the issue?

Reply:
https://gitlab.gnome.org/GNOME/glib/-/issues/443